### PR TITLE
Fix documentation and formatting issues

### LINKED ISSUE #9706: Documentation 2022b
Issue overview
--------------
A place to keep an ongoing list of documentation issues.

### PR DESCRIPTION
**Pull request by yujiex**
Pull request overview
---------------------
 - Fixes [#9706](https://github.com/NREL/EnergyPlus/issues/9706)

Three documentation typo/formatting issues are to be fixed
- [x] Field: Disctrict Cooling COP has a typo, see https://bigladdersoftware.com/epx/docs/22-2/input-output-reference/input-for-output.html#environmentalimpactfactors
- [ ] Units for m^3 not rendering correctly in EMS Application Guide's Table 2.7 for [Built-In Psychrometric Functions](https://bigladdersoftware.com/epx/docs/22-2/ems-application-guide/built-in-functions.html#built-in-psychrometric-functions), looks like an errant use of < sup> < /sup >. Didn't find this in other parts of the EMS Application Guide.
- [x] Equation formatting issue at the bottom of the [Sky Radiation Modelling](https://bigladdersoftware.com/epx/docs/22-1/engineering-reference/climate-calculations.html#sky-radiation-modeling) section in the engineering reference.

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: doc/ems-application-guide/src/energyplus-runtime-language/built-in-functions.tex [hunk 1]
```diff
@@ -118,7 +118,7 @@ \subsection{Built-in Psychrometric Functions}\label{built-in-psychrometric-funct
 \midrule
 \endhead
 
-@RhoAirFnPbTdbW & Result & Density of moist air & kg/m < sup > 3 < /sup > \tabularnewline
+@RhoAirFnPbTdbW & Result & Density of moist air & \si{\kg\per\m\cubed} \tabularnewline
```
Comment (mjwitte) [inline doc/ems-application-guide/src/energyplus-runtime-language/built-in-functions.tex:121]: @yujiex There are more places in the same table that need this fix (or similar). RhovFnTdbRh, RhovFnTdbWPb, VFnTdbWPb, and RhoH2O.

Comment (yujiex) [reply doc/ems-application-guide/src/energyplus-runtime-language/built-in-functions.tex:121]: My bad. Will fix them too.

**Comment by mjwitte** [comment]
Also, when I build the I/O Ref locally, there are several warnings like this:
```
1>Building Custom Rule C:/MJWGit/EnergyPlus-Review1/doc/input-output-reference/CMakeLists.txt
1>[LATEXERROR::doc\input-output-reference\src\overview\group-heating-and-cooling-coils.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
1>[LATEXERROR::doc\input-output-reference\src\overview\group-heating-and-cooling-coils.tex:None::Undefined control sequence.\per ->\ERROR]
1>[LATEXERROR::doc\input-output-reference\src\overview\group-heating-and-cooling-coils.tex:None::Undefined control sequence.\watt ->\ERROR]
```
It would be great if you can find these and fix them.

**Comment by mjwitte** [comment]
Similar warnings are reported for `group-evaporative-coolers.tex`.

**Comment by yujiex** [comment]
> Also, when I build the I/O Ref locally, there are several warnings like this:
> 
> ```
> 1>Building Custom Rule C:/MJWGit/EnergyPlus-Review1/doc/input-output-reference/CMakeLists.txt
> 1>[LATEXERROR::doc\input-output-reference\src\overview\group-heating-and-cooling-coils.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
> 1>[LATEXERROR::doc\input-output-reference\src\overview\group-heating-and-cooling-coils.tex:None::Undefined control sequence.\per ->\ERROR]
> 1>[LATEXERROR::doc\input-output-reference\src\overview\group-heating-and-cooling-coils.tex:None::Undefined control sequence.\watt ->\ERROR]
> ```
> 
> It would be great if you can find these and fix them.

Sure. I'll look into these as well

**Comment by yujiex** [comment]
Hi @mjwitte, I tried building on my end (with DOCUMENTATION_BUILD flag turned on in cmake). I don't see warning messages like those. The pdf also seems to render fine. 
image

I noticed on the bigladder website, the m^3/s is not rendered correctly for the same paragraphs as the above.
image

I'm not sure how to build the html version of the documentation. Were you building the html ones? Would you mind showing me how to do that? Thanks.

**Comment by mjwitte** [comment]
I'm building pdf.

The error in group-heating-and-cooling-coils.tex is [here](Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed) for `Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed`. The SI units in these two field descriptions are blank, just rendering (). These happen to be the only place in the entire documentation source that use `\volumeFlowRateperWatt`. 
image

The problem is with this header.tex definition (and other similar ones). I'm working on a solution (trial and error).
`\DeclareSIUnit\volumeFlowRateperWatt{({\volumeFlowRate})\per\watt}`

**Comment by nealkruis** [comment]
@yujiex I'll have my team look into the HTML issue. @aaron-boranian @michael-okeefe

**Comment by mjwitte** [comment]
FWIW, the units for volumeFlowRateperWatt worked fine up until v9.6. Probably some LaTeX software update broke it? Anyways, not worth obsessing over. I'll push a fix shortly. These aren't exactly what they used to be, but they're good enough.
v9.2  image
this branch image
v9.2 image
this branch image

File: doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex [hunk 2]
```diff
@@ -7404,11 +7404,11 @@ \subsubsection{Inputs}\label{inputs-31}
 
 \paragraph{Field: Speed \textless{}x\textgreater{} Reference Unit Rated Air Flow Rate}\label{field-speed-x-reference-unit-rated-air-flow-rate-2}
 
-This numeric field defines the volumetric air flow rate, in m\(^{3}\)per second, across the cooling coil at rated conditions for Speed \textless{}x\textgreater{} operation. The value entered here should be directly from the Reference Unit data, corresponding to the given gross rated total cooling capacity and gross rated cooling COP at the speed, as above.
+This numeric field defines the volumetric air flow rate, in m\(^{3}\) per second, across the cooling coil at rated conditions for Speed \textless{}x\textgreater{} operation. The value entered here should be directly from the Reference Unit data, corresponding to the given gross rated total cooling capacity and gross rated cooling COP at the speed, as above.
```
Comment (mjwitte) [inline doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex]: Ideally, these should be updated to use the appropriate `\si` macros, but I won't hold this up for that.

Comment (yujiex) [reply doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex]: Got you, I can make these changes.

Also previous part I thought the error was from `\volumeFlowRate`. Will check out the `\volumeFlowRateperWatt` @mjwitte

**Comment by yujiex** [comment]
> @yujiex I'll have my team look into the HTML issue. @aaron-boranian @michael-okeefe

Thanks @nealkruis

**Review by mjwitte — APPROVED** [review]
@yujiex Looks good. Thanks.

**Pull request merged**